qt/applications/workbench/workbench/plotting/propertiesdialog.py

```diff
@@ -173,7 +173,7 @@ def __init__(self, canvas, axes):
 
 
 class MarkerEditor(QWidget):
-    def __init__(self, filename, valid_style, valid_colors):
+    def __init__(self, filename, valid_style, valid_colors, used_names=[]):
```

Comment:
It is not best practice to use [mutable objects as defaults](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) in Python. Can we follow the standard guidance as suggested on the guide and use `None`?